test/libyul/ControlFlowGraphTest.cpp

```diff
@@ -41,7 +41,7 @@ using namespace solidity::frontend;
 using namespace solidity::frontend::test;
 
 ControlFlowGraphTest::ControlFlowGraphTest(std::string const& _filename):
-	TestCase(_filename)
+	frontend::test::EVMVersionRestrictedTestCase(_filename)
```

Comment:
```suggestion
	EVMVersionRestrictedTestCase(_filename)
```

Same with other uses in files below.

Reply:
You still have it in those other files I mentioned.

Also redundant `yul::` in `ControlFlowGraphBuilder.cpp`.